pkg/controller/instance/plan_execution_engine.go

```diff
@@ -254,21 +267,21 @@ func prepareKubeResources(plan *activePlan, meta *executionMetadata, renderer ku
 						if resource, ok := plan.Templates[res]; ok {
 							templatedYaml, err := engine.Render(resource, configs)
 							if err != nil {
-								phaseState.State = v1alpha1.PhaseStateError
-								stepState.State = v1alpha1.PhaseStateError
+								phaseState.Status = v1alpha1.ExecutionFatalError
+								stepState.Status = v1alpha1.ExecutionFatalError
 
-								err := errors.Wrapf(err, "error expanding template")
+								err := errwrap.Wrapf(err, "error expanding template")
```

Comment:
use Wrap and not Wrapf if you're not doing any format templating

Reply:
also, rename your qualified import here after the package, not after the what you're using. :) For this particular package, I've used `pkgerrors` before since it's about as literal as it could be (github.com/pkg/errors)